packages/@react-spectrum/s2/src/SelectBoxGroup.tsx

```diff
@@ -0,0 +1,446 @@
+  let {children, value, isDisabled: individualDisabled = false, UNSAFE_style} = props;
+  
+  let {
+    orientation = 'vertical',
+    isDisabled: groupDisabled = false,
+    showCheckbox = false
+  } = useContext(SelectBoxContext);
+
+  const size = 'M';
+  const isDisabled = individualDisabled || groupDisabled;
+
+  return (
+    <ListBoxItem
+      id={value}
+      isDisabled={isDisabled}
+      textValue={value}
+      className={renderProps => (props.UNSAFE_className || '') + selectBoxStyles({
+        size, 
+        orientation, 
+        ...renderProps
+      }, props.styles)}
+      style={UNSAFE_style}>
+      {(renderProps) => (
+        <>
+          {showCheckbox && (renderProps.isSelected || (!renderProps.isDisabled && renderProps.isHovered)) && (
+            <div 
+              className={style({
+                position: 'absolute',
+                top: 12,
+                left: 12
+              })}
+              aria-hidden="true">
+              <div
+                className={box({
+                  isSelected: renderProps.isSelected,
+                  isDisabled: renderProps.isDisabled,
+                  size: 'M'
+                } as any)}>
+                {renderProps.isSelected && (
+                  <Checkmark 
+                    size="S" 
+                    className={iconStyles} />
+                )}
+              </div>
+            </div>
+          )}
+          <Provider
+            values={[
+              [IllustrationContext, {
+                size: 'S',
+                styles: illustrationContainer({size: 'S', orientation, isDisabled: renderProps.isDisabled})
+              }],
+              [TextContext, {
+                slots: {
+                  [DEFAULT_SLOT]: {
+                    styles: labelText({orientation, isDisabled: renderProps.isDisabled})
+                  },
+                  label: {
+                    styles: labelText({orientation, isDisabled: renderProps.isDisabled})
+                  },
+                  description: {
+                    styles: descriptionText({orientation, isDisabled: renderProps.isDisabled})
+                  }
+                }
+              }]
+            ]}>
+            {children}
+          </Provider>
+        </>
+      )}
+    </ListBoxItem>
+  );
+}
+
+/**
+ * SelectBoxGroup allows users to select one or more options from a list.
+ */
+export const SelectBoxGroup = /*#__PURE__*/ forwardRef(function SelectBoxGroup<T>(props: SelectBoxGroupProps<T>, ref: DOMRef<HTMLDivElement>) {
+  [props, ref] = useSpectrumContextProps(props, ref, SelectBoxGroupContext);
+
+  let {
+    children,
+    onSelectionChange,
+    selectedKeys: controlledSelectedKeys,
+    defaultSelectedKeys,
+    selectionMode = 'single',
+    orientation = 'vertical',
+    numColumns = 2,
+    gutterWidth = 'default',
+    isDisabled = false,
+    showCheckbox = false,
+    UNSAFE_className,
+    UNSAFE_style
+  } = props;
```

Comment:
```suggestion
  let {
    children,
    onSelectionChange,
    selectedKeys: controlledSelectedKeys,
    defaultSelectedKeys,
    selectionMode = 'single',
    orientation = 'vertical',
    numColumns = 2,
    gutterWidth = 'default',
    isDisabled = false,
    showCheckbox = false,
    UNSAFE_className,
    UNSAFE_style,
    ...otherProps
  } = props;
```

then you can spread them on ListBox like Daniel suggested